ui/tests/helpers/secret-engine/secret-engine-selectors.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const SECRET_ENGINE_SELECTORS = {
+  backButton: '[data-test-back-button]',
+  configTab: '[data-test-configuration-tab]',
+  configure: '[data-test-secret-backend-configure]',
+  configureTitle: (type: string) => `[data-test-backend-configure-title="${type}"]`,
+  configurationToggle: '[data-test-mount-config-toggle]',
+  createSecret: '[data-test-secret-create]',
+  crumb: (path: string) => `[data-test-secret-breadcrumb="${path}"] a`,
+  generateLink: '[data-test-backend-credentials]',
+  mountType: (name: string) => `[data-test-mount-type="${name}"]`,
+  mountSubmit: '[data-test-mount-submit]',
+  secretHeader: '[data-test-secret-header]',
+  secretLink: (name: string) => (name ? `[data-test-secret-link="${name}"]` : '[data-test-secret-link]'),
+  ttlToggle: '[data-test-ttl-toggle="TTL"]',
```

Comment:
mostly an FYI this selector already exists in https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/components/ttl-picker-selectors.ts#L17-L18